var/spack/repos/builtin/packages/dialign-tx/package.py

```diff
@@ -50,3 +50,5 @@ def install(self, spec, prefix):
             install('dialign-tx', prefix.bin)
             # t-coffee recognizes as dialign-t
             install('dialign-tx', join_path(prefix.bin, 'dialign-t'))
+
+    patch('dialign-1-0-2-gcc-5-4-0.patch', when='%gcc@5.4.0')
```

Comment:
please add a comment where this patch is coming from!

Reply:
I started with this report of changes found necessary on a debian platform to use a different compiler entitled: "Debian Bug report logs - #757281 dialign-t: FTBFS with clang instead of gcc" (https://bugs.debian.org/cgi-bin/bugreport.cgi?bug=757281;msg=7).